tests/unit_tests/test_node_logger.py

```diff
@@ -0,0 +1,37 @@
+from mipengine.node import node_logger
+from mipengine.node.node_logger import initialise_logger
+from mipengine.node import config as node_config
+from unittest.mock import patch
+
+import pytest
+
+from mipengine import AttrDict
+from mipengine.node import node_logger
+
+
+@pytest.fixture(scope="module", autouse=True)
+def mock_node_config():
+    node_config = AttrDict(
+        {
+            "identifier": "localnode1",
+            "log_level": "INFO",
+            "role": "LOCALNODE",
+        }
+    )
+
+    with patch(
+        "mipengine.node.node_logger.node_config",
+        node_config,
+    ):
+        yield
+
+    return node_config
+
+
+def test_get_ctx_id_from_args():
```

Comment:
I was checking that it get the context id from the arguments correctly, because of the if condition in my decorator. I can remove it if it feels unnecessary, no problem

Reply:
Ah okay, makes sense!

Are you checking the output? It appears to me that you are initializing the logger but not checking if its initialized properly. Am I seeing something wrong?